sys/compat/linuxkpi/common/src/linux_compat.c

```diff
@@ -1993,17 +1993,17 @@ iounmap(void *addr)
 void *
 vmap(struct page **pages, unsigned int count, unsigned long flags, int prot)
 {
-	vm_pointer_t off;
```

Comment:
It would be useful to upstream this commit (renaming the variable) to keep our local diff down and reduce the chance for conflicts in the future.  A more typical name in the FreeBSD kernel for this would probably be `va` rather than `vaddr` FWIW (especially in sys/vm).

Reply:
Ok renamed.